gtfs/spec/en/reference.md

```diff
@@ -413,6 +419,85 @@ Trip planners normally calculate transfer points based on the relative proximity
+|   |  | * **7** - Exit gate. Indicates a pathway exiting an area where proof-of-payment is required into an area where proof-of-payment is no longer required. |
+|  pathway_type | Optional | The **pathway_type** field specifies the type of connection. Valid values for this field are: |
+|   |  | * **1** - Connects station locations to an entrance. |
+|   |  | * **2** - Connects stops/stations for transfers off the streets or within stations. |
+|   |  | * **3** - Connects stops/stations/entrances for transfers on the streets. |
+|  traversal_time | Optional | The **traversal_time** field specifies the time needed to go through the pathway on foot. Valid values for this field are: |
+|   |  | * **(empty)** - Unknown travel time. |
+|   |  | * **(non-negative integer)** - Number of seconds needed to travers this pathway on foot. |
+|  wheelchair_traversal_time | Optional | The **wheelchair_traversal_time** field specifies the time needed to go through the pathway in a wheelchair. Valid values for this field are: |
+|   |  | * **(empty)** - Unknown travel time. |
+|   |  | * **-1** - Pathway not wheelchair accessible. |
+|   |  | * **(non-negative integer)** - Number of seconds needed to travers this pathway in a wheelchair. |
+|  ramp_slope | Optional | The **ramp_slope** field specifies the slope ratio of the pathway. Valid values for this field are: |
+|   |  | * **(empty)** - Unknown slope. |
+|   |  | * **(float)** - Slope ratio of the pathway. |
+|   |  | Note: This field is allowed only for pathway_mode 1 (walkway) and 3 (travelator). |
+|  stair_count | Optional | The **stair_count** field specifies the number of stairs of the pathway. Valid values for this field are: |
+|   |  | * **(empty)** - Unknown number of stairs. |
+|   |  | * **(non-negative integer)** - Total number of stairs of the pathway. |
+|   |  | Note: This field is allowed only for pathway_mode 2 (stairs) and 4 (escalator). |
+|   |  | Note: The approximation of 1 floor = 15 stairs (or 12 for an escalator) can be used to generate approximative values. |
+|  pathway_name | Optional | The **pathway_name** field contains the name of the pathway, if any. Please use a name that people will understand in the local and tourist vernacular. |
+|  pathway_code | Optional | The **pathway_code** field contains a short text or a number that uniquely identifies the pathway for passengers. The **pathway_code** can be the same as pathway_id if it is passenger-facing. This field should be left blank for pathway without a code presented to passengers. Example: Elevator “A”. |
+|  signposted_as | Optional | The **signposted_as** field helps to generate schematic directions like 'follow signs to <signposted as>' where <signposted as> part represents specific text on the signs that should remain unaffected by translation into other languages. |
```

Comment:
Did you mean "semantic directions" rather than "schematic directions" here?

Reply:
@dbabramov: This question is for you.

Reply:
Apologies, I am not a native speaker, so writing well-formed English sentences can be quite a challenge for me.
A native speaker I asked to take a look at this sentence suggested ditching the whole thing in favour of:
"The **signposted_as** field is an exact string of text from physical signage visible to transit riders. The string can be used to provide textual directions to users, such as 'follow signs to <signposted as>'. The language as printed on the signs should not be translated in this field."

Reply:
Updated.